File: benchmark/results/ts-monolith-opus-script/run-002/b-baseline/output/src/orders/services/OrderService.ts
import { Order } from "../aggregates/Order";
import { Id } from "../../shared/value-objects/Id";

/** CRUD operations for orders. */
export class OrderService {
  create(input: Partial<Order>): Order {
    void input;
    throw new Error("not implemented");
  }

  findById(id: Id): Order | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): Order[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: Id): void {
    void id;
    throw new Error("not implemented");
  }
}

File: benchmark/results/java-invocation-script/run-001/b-baseline/output/com/metaengine/demo/ordering/services/OrderService.java
package com.metaengine.demo.ordering.services;

import com.metaengine.demo.ordering.aggregates.Order;
import java.util.List;

/** OrderService for the ordering domain. */
public class OrderService {

    /** Create a new Order. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Order by id; may return null. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List orders up to the given limit. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an order by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
